Makefile

```diff
@@ -61,7 +61,7 @@ endif
 prove: HARNESS ?= TAP::Harness
 prove: PROVE_FILES ?= $(shell find t examples  -type f -name "*.t")
 prove: export TEST_NGINX_RANDOMIZE=1
-prove: $(ROVER) nginx cpan ## Test nginx
+prove: $(ROVER) nginx cpan ## Test nginx. To execute a specific test: make prove PROVE_FILES=t/testname.t
```

Comment:
Here to. I don't intend `PROVE_FILES` to be documented and used by public.

Reply:
done in 72f9a669a6fae2a9be05f3c9faa03060dc1ee5e7